controllers/volume.go

```diff
@@ -131,6 +131,9 @@ func UpdateVolume(c *gin.Context) {
 		return
 	}
 
+	// Invalidate volume in transport
+	models.Transport.ActiveVolumes.RemoveEnqueuedInstance(volumeUUID)
```

Comment:
a czemu go usuwamy, skoro poprawiliśmy jego dane? (bo wzięliśmy wskaźnik do tego volume poprzez transport.getVolume(..),  prawda?

Reply:
To prawda, poprawiliśmy jego dane, ale żeby ten Volume poprawnie działał z nowymi ustawieniami musielibyśmy jeszcze od nowa go zinicjalizować, przynajmniej w kontekście usług, np. stworzyć i zinicjalizować nowy partitioner, zinicjalizować nową metodę backupu czy szyfrowania. Moim zdaniem łatwiej jest to zrobić w momencie gdy volume będzie na nowo powstawał niż implementować te funkcje drugi raz przy update. 

Z drugiej strony zinicjalizowanie np. throughput partitionera zajmuje trochę czasu (bo wymaga sprawdzenia uploadu i downloadu), moim zdaniem lepiej byłoby to zrobić przy następnym faktycznym użyciu volume do operacji dyskowych (gdy zostanie od nowa stworzony w transporcie) niż w momencie update, gdzie może później nie być operacji dyskowych.